sdk/sas-auth-browser/examples/basic-example.html

```diff
@@ -10,66 +10,74 @@
     <script src="https://cdn.developer.sas.com/packages/sas-auth-browser/latest/dist/index.js"></script>
     <script>
       (async () => {
-        const viyaUrl = "YOUR-VIYA-URL";
+        const content = document.getElementById('content');
```

Comment:
This mostly a format change with a slight refactor to include `logout()` in the example.